src/fixes.rs

```diff
@@ -0,0 +1,147 @@
+use crate::common::*;
+
+mod lowercase_key;
+
+trait Fix {
+    fn name(&self) -> &str;
+
+    fn fix_warnings(
+        &self,
+        warnings: Vec<&mut Warning>,
+        lines: &mut Vec<LineEntry>,
+    ) -> Option<usize> {
+        let mut count: usize = 0;
+        for warning in warnings {
+            let line = &mut lines[warning.line_number() - 1];
+            if self.fix_line(line).is_some() {
+                warning.set_fixed(true);
+                count += 1;
+            }
+        }
+
+        Some(count)
+    }
+
+    fn fix_line(&self, _line: &mut LineEntry) -> Option<()> {
+        None
+    }
+
+    fn set_fixed_on_all(&self, warnings: Vec<&mut Warning>) -> Option<usize> {
```

Comment:
This method is not used?
If so, please remove it.

Reply:
Yes, this is remains from some of the experiments )

Reply:
I removed it.